src/views/Lottery/index.tsx

```diff
@@ -76,7 +76,12 @@ const Lottery = () => {
       <PageSection background={CHECK_PRIZES_BG} hasCurvedDivider={false} index={2}>
         <CheckPrizesSection />
       </PageSection>
-      <PageSection background={isDark ? FINISHED_ROUNDS_BG_DARK : FINISHED_ROUNDS_BG} concaveDivider index={2}>
+      <PageSection
+        innerProps={{ style: { margin: '0', width: '100%' } }}
```

Comment:
Make the outer container fill the width, regardless of content (prevents card changing width dramatically when details are expanded/closed)